docs/source/03_tutorial/06_visualise_pipeline.md

```diff
@@ -96,3 +96,72 @@ kedro viz --load-file my_shareable_pipeline.json
 ```
 
 And this will visualise the pipeline visualisation saved as `my_shareable_pipeline.json`.
+
+
+## Visualise Plotly charts in Kedro-viz
+
+[Plotly](https://plotly.com/python/) is a free and open source python library that allows you to make interactive, publication quality graphs. With Plotly integration on Kedro-viz, you can output your interactive charts as part of your pipline visualisation. 
```

Comment:
```suggestion
[Plotly](https://plotly.com/python/) is a free and open source python library that allows you to make interactive, publication quality graphs. With the Plotly integration on Kedro-Viz, you can output your interactive charts as part of your pipeline visualisation. 
```

Reply:
I think _Python_ should be capitalized, like it is in the readme.